### DISCUSSION TITLE
Atualizei o RMM para v0.19.1 e não consigo mais abrir os agentes.

### DISCUSSION THREAD

**Discussion by Cleberson-Brandao** [amidaware/tacticalrmm#2038] [General]
I updated the SSL certificate and also the RMM version to v0.19.1, but now I can't connect to the machines, the connect button is gone, can anyone help me?

**Comment by dinger1986** [comment]
Did your certs renew properly? What version did you upgrade from?

**Comment by Cleberson-Brandao** [comment]
I updated the certificate today, the current version is v0.19.1.

**Comment by dinger1986** [comment]
What command did you run to update the certs?

What version did you update from?

**Comment by Cleberson-Brandao** [comment]
sudo certbot certonly --manual -d *.example.com --agree-tos --no-bootstrap --preferred-challenges dns -m admin@example.com --no-eff-email

with changes to my domain.

I deployed the DNS TXT, waited for it to propagate to everyone, validated it on https://www.whatsmydns.net/#TXT/...

then I ran the command,
./update.sh --force

**Comment by dinger1986** [comment]
What version were you on before?

What's the output on ls /etc/letsencrypt/live

**Comment by Cleberson-Brandao** [comment]
v0.17.

**Comment by dinger1986** [comment]
Can you please read all I have written and reply to the whole thing. I'm trying to help you

**Comment by Cleberson-Brandao** [comment]
![Captura de tela 2024-07-19 160012](https://github.com/user-attachments/assets/a8ee0fed-e40f-4efa-b455-e9745f148687)

**Comment by dinger1986** [comment]
Are you doing anything unsupported?

I can't access rmm.yourdomain.com

**Comment by Cleberson-Brandao** [comment]
The Analyst implemented the system is not present, he was delegated to update the SSL certificate, and I followed the step by step according to the manual, however after the update the take control service is not appearing.

**Comment by Cleberson-Brandao** [comment]
![Captura de tela 2024-07-19 160957](https://github.com/user-attachments/assets/fa219d22-7798-4729-afa9-d6aafbdecd16)

**Comment by dinger1986** [comment]
But your system, is it setup supported?

Any reasons why I can access it from my office?

Maybe worth getting in touch with amidaware for commercial support if you don't know how it's setup https://docs.tacticalrmm.com/support/#commercial-support
